documentation/src/main/asciidoc/querylanguage/Concepts.adoc

```diff
@@ -689,7 +689,7 @@ for (var map : results) {
 [%unbreakable]
 ----
 var results =
-        entityManager.createQuery("select title, left(book.text, 200) from Book",
+        entityManager.createQuery("select title, left(text, 200) from Book",
```

Comment:
I tested locally and found either we need to add `book` alias or simply remove as above; otherwise the queres are invalid.